_includes/top-navigation.html

```diff
@@ -8,7 +8,7 @@
                         <span class="fa fa-bars"></span>
                     </button>
                     <a class="navbar-brand" href="{{ site.baseurl }}/">
-                        <img id="logo-header" src="{{ site.baseurl }}/concur_assets/img/Concur_Logo_VT_Color_40px.png" alt="Logo">
+                        <img id="logo-header" src="{{ site.baseurl }}/concur_assets/img/concur_logo.gif" alt="Logo">
```

Comment:
does this work out to be the same dimensions as the current, or does it change the overall header dimensions?